test/tests-events.js

```diff
@@ -65,15 +68,22 @@ test('once', async (t) => {
   await sleep(5000)
 })
 
-test.failing('on - Wildcard', async (t) => {
+test('on - Wildcard', async (t) => {
   const client = connect('emit-on-wildcard')
 
+  t.plan(5)
```

Comment:
this fails, with 6 asserts.
It is weak to plan for a number of message received.
Instead, test exactly the 3 message are well received: hello, again, and...